packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx

```diff
@@ -363,11 +367,11 @@ const ConditionsWidget: React.FC<ConditionsWidgetProps> = ({
   );
 };
 
-function RequiredLabel({ isRequired, text, t }: RequiredLabelProps) {
+function RequiredFieldLabel({ editing, label, t }) {
   return (
     <>
-      <span>{text}</span>
-      {isRequired && (
+      <span>{label}</span>
+      {!editing && (
```

Comment:
I'm not sure I quite understand the reason for changing things based on whether someone is editing or not? It seems like a required field is required regardless and should be marked that way no?

Reply:
That's right, @ibacher. A bad rebase was the culprit here.